packages/ts-client/src/webrtc/tracks/LocalTrack.ts

```diff
@@ -284,3 +288,30 @@ export class LocalTrack<EndpointMetadata, TrackMetadata> implements TrackCommon
     });
   };
 }
+
+function emitEvents<EndpointMetadata, TrackMetadata>(
```

Comment:
WDYT about having separate file with event related logic?

Reply:
I placed it here because I wanted to avoid exporting functions that have only one import from a single file, so they are basically private.

But I moved them :)